### PR TITLE
Add annotations to schema

### PR DESCRIPTION
**Pull request by willm30**
## Description

This PR adds the annotation model to the schema, and wires annotations into the `mintTokens` saga to test.

It does not add ipfs. See https://github.com/JoinColony/colonyCDapp/pull/902

Test with: https://github.com/JoinColony/colonyCDapp/pull/909

### CODE REVIEW

File: amplify/backend/api/colonycdapp/schema.graphql [hunk 1]
```diff
@@ -1490,11 +1490,8 @@ type ColonyFundsClaim @model {
 """
 Represents a native Colony Chain Funds Claim
 E.g., Token 0x0000...0000: ETH, xDAI, etc
-""" # This is really not elegant, but there's no other proper solution, as you can't
-type # return the Token @hadOne relationship from the lambda function
-# Note that we don't need token details for the values that we're fetching (since
-# it's implied), but it would have been nice not to have to basically repeat the type
-ColonyChainFundsClaim {
+""" # This is really not elegant, but there's no other proper solution, as you can't # return the Token @hadOne relationship from the lambda function # Note that we don't need token details for the values that we're fetching (since # it's implied), but it would have been nice not to have to basically repeat the type
```
Comment (thomassdk) [inline amplify/backend/api/colonycdapp/schema.graphql:1493]: why have these separate lines all been combined into one?

if there a single line is correct should we remove these extraneous # symbols?

File: src/redux/sagas/actions/mintTokens.ts [hunk 2]
```diff
@@ -113,21 +120,39 @@ function* createMintTokensAction({
...
+      // const ipfsHash = yield call(uploadIfpsAnnotation, annotationMessage);
+
+      yield apolloClient.mutate<
+        CreateAnnotationMutation,
+        CreateAnnotationMutationVariables
+      >({
+        mutation: CreateAnnotationDocument,
+        variables: {
+          input: {
+            message: annotationMessage,
```
Comment (thomassdk) [inline src/redux/sagas/actions/mintTokens.ts:135]: I believe this is error is showing up as the `generated.ts` is different from the schema

**Review by thomassdk — APPROVED** [review]
LGTM

File: amplify/backend/api/colonycdapp/schema.graphql (hunk 1 above)
Comment (willm30) [reply amplify/backend/api/colonycdapp/schema.graphql:1493]: Don't know, it's weird and potentially prettier related.

**Pull request merged**